StoreApp/Controllers/AccountController.cs

```diff
@@ -212,18 +213,21 @@ public ActionResult EditUserAdmin(UserViewModel user)
         {
             if ((Session["Role"] as string).Equals(UserRoles.admin.ToString()))
             {
-                user.Role = user.Role.Equals((int)UserRoles.user) ? UserRoles.user.ToString() 
-                                                                  : UserRoles.admin.ToString();
+                var i = UserRoles.user.ToString();
```

Comment:
Why do you need to set this variable? What's the reason?

Reply:
fixed